framework/src/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -1539,6 +1589,34 @@ public static Range getRange(AnnotationMirror rangeAnno) {
     }
 
     /**
+     * Returns a {@code Range} bounded by the values specified in the given {@code @IntRange} or
+     * {@code @IntVal} annotation. Returns null if the given annotations is not {@code IntRange} or
+     * {@code IntVal}.
```

Comment:
Why does this method not need to cover `@IntRangeFromPositive`?
Why is null allowed for getRange but not for this method?